src/ValidationError.js

```diff
@@ -20,11 +20,29 @@ class ValidationError extends Error {
     });
 
     this.errors.forEach((err) => {
-      this.message += `options${err.dataPath} ${err.message}\n`;
+      this.message += ValidationError.format(err);
     });
 
     Error.captureStackTrace(this, this.constructor);
   }
+
+  static format(err) {
+    const {
+      keyword,
+      message,
+      dataPath,
+      params: { additionalProperty },
+    } = err;
+    const errorMessage = `options${dataPath} ${message}\n`;
+
+    if (keyword === 'additionalProperties') {
+      return `options${err.dataPath} ${
```

Comment:
If I remember right the `dataPath` is not defined for `additionalProperties` (`err.dataPath`)? So we might use `additionalProperty` here instead e.g

### Default

```
options${additionalProperty} is NOT a valid options property
```

### Customized

```
options${additionalProperty} is NOT a valid options property (https://github.com/org/repo#options)
```